tests/common.go

```diff
@@ -1904,6 +1912,58 @@ func ValidateApplications(contexts []*scheduler.Context) {
 	})
 }
 
+// ValidateApplicationsStartData validates applications and start continous data injection to the same
+func ValidateApplicationsStartData(contexts []*scheduler.Context, appContext context1.Context) (chan string, *errgroup.Group, map[string][]appDriver.ApplicationDriver) {
```

Comment:
Can we think of a better way to return values? Taking 2 inputs and returning 3 values is not ideal

Reply:
As discussed removed map[string][]appDriver.ApplicationDriver